dwitter/feed/views.py

```diff
@@ -92,6 +91,12 @@ def feed(request, page_nr, sort):
         .select_related('reply_to__author__username')
         .prefetch_related('comments'))
 
+    # For some reason order_by('?') and .annotate(num_likes=Count('likes'))
+    # don't work together, so we need to do extra work if the sorting is random.
```

Comment:
This line is a single character too long, so the lint and build fails. FIX IT! :)

Reply:
The 80 character limit should not be a hard limit IMO. Fun fact: The 80 character "limit" is based on the legacy of the 80 column punched card format

Reply:
While I tend to agree I also think it's alright to have a strict build system based on the linter. We could change the lint config though